Dockerfile.frontend-base

```diff
@@ -32,8 +32,16 @@ RUN chown -R frontend /obs/src/api
 # builds this container
 USER frontend
 WORKDIR /obs/src/api
-RUN export NOKOGIRI_USE_SYSTEM_LIBRARIES=1; bundle install --jobs=3 --retry=3
 
+# Set up some convenience ruby binaries...
+RUN for i in ruby gem irb; do ln -s /usr/bin/$i.ruby2.4 ~/bin/$i; done
+
+# Install our bundle
+# FIXME: Retrying bundler if it fails is a workaround for https://github.com/moby/moby/issues/783
+#        which seems to happen on openSUSE (< Tumbleweed 20171001)...
+RUN export NOKOGIRI_USE_SYSTEM_LIBRARIES=1; bundle install --jobs=3 --retry=3 || bundle install --jobs=3 --retry=3
```

Comment:
--retry=6 would have different results I assume?

Reply:
Yeah this is not about bundler, this is about the filesystem used by docker...